tools/kete.py

```diff
@@ -543,13 +543,19 @@ def _log_drawings_available(self, device):
 
 
 class Fetcher(Worker):
-    def __init__(self, manager, args):
+    def __init__(self, manager, args, config):
         super(Fetcher, self).__init__(manager)
         self.device = None
         self.timestamps = None
         address = args.address
         index = args.index
 
+        if not address in config:
+            config[address] = {}
```

Comment:
`tuhi` has per BT address directories with `settings.ini` instead of one gigantic `settings.ini`. Not sure which one is the best given the few options we have to configure

Reply:
I think most people have at most 2 tablets. I don't see the interest of creating multiple directory for a not so common use-case. What do you think ?

Reply:
Problem is, I do not recall the rational behind the separate dir for tablets in tuhi. I guess we can keep the single file for now and decide what to do before making a new release.

At least we can get @whot's opinion when he is back.